package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@nextcloud/cypress",
-  "version": "1.0.0",
```

Let's release 1.0.0 first? xD

---

Also, maybe as beta.0 ?
```suggestion
  "version": "1.0.0-beta.0",
```

---

Already did but without build files :see_no_evil:

---

Need a new tag now

---

Why is this merged?